src/views/v2/tickets/item/content/title.php

```diff
@@ -36,7 +36,7 @@
 <div <?php tribe_classes( $title_classes ); ?> >
 	<?php if ( $is_mini ) : ?>
 		<div <?php tribe_classes( $event_title_classes ); ?> >
-			<?php echo esc_html( get_the_title( $post_id ) ); ?>
+			<?php echo get_the_title( $post_id ); ?>
```

Comment:
:no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'get_the_title'.

<sub>hash: 09202ed14cdd0f25c2329d0c646b6161</sub>

Reply:
@sc0ttkclark ☝️

Reply:
I think we should want to go with `wp_kses_post()` here.

Reply:
@sc0ttkclark said no in the Slack DM I double-checked him with. We'll see; maybe he was tired.

Reply:
Basically the_title() doesn't do anything extra, so there's an intention in regards to titles with their rendering.